src/CommandQueue.ts

```diff
@@ -47,6 +56,20 @@ export class CommandQueue {
         registerDomEvents: (payload: any): void => {
             this.orchestration.registerDomEvents(payload);
         },
+        recordEvent: (payload: RecordEventInput) => {
```

Comment:
issue: If we're going to check the payload for type safety, the proper type is `unknown` (indicates the type could be anything but is the opposite of `any` in that `unknown` values cannot be assigned to any other type).